pandas_gbq/load.py

```diff
@@ -99,7 +99,15 @@ def cast_dataframe_for_parquet(
                 errors="ignore",
             )
         elif column_type in {"NUMERIC", "DECIMAL", "BIGNUMERIC", "BIGDECIMAL"}:
-            cast_column = dataframe[column_name].map(decimal.Decimal)
+            # decimal.Decimal does not support `None` input, add support here.
+            # https://github.com/googleapis/python-bigquery-pandas/issues/719
+            def convert(x):
+                if x is None:
```

Comment:
QUESTION:

The [original issue](https://github.com/googleapis/python-bigquery-pandas/issues/719) mentions that `None` and `pd.NA` are not handled appropriately.

This code seems to only address `None`.
Is the intent to check for both?

IF yes... this may be a possible check that is more general and as other edge cases arise could enable edits to the code.
```suggestion
                if x in {None, pd.NA}:
```

Reply:
Indeed, thank you for catching this! I'll update the code.